tests/tests/common_update.py

```diff
@@ -13,15 +13,17 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 
-from ..common_docker import *
+import tempfile
+import random
+
+from fabric.api import *
+import pytest
+
+from .. import conftest
 from ..common import *
```

Comment:
left on purpose?

Reply:
I mean I see the commit message, but why everything else and not this?

Reply:
Yes, see commit comment: https://github.com/mendersoftware/integration/pull/742/commits/9b6aba7acf92911208a0d4a7ae9fc5f3e8f22af1

Basically the common is overriding some fabric functions, so it must be this way (for now, at least).

Reply:
I should have been explicit in the commit message, you are right!